.travis.yml

```diff
@@ -4,7 +4,10 @@ os:
   - linux
   - osx
```

Comment:
```suggestion
```

I highly doubt this package behaves differently in different OSes. So, how about not testing with macOS to speedup CI?

Reply:
:+1: I also think 1.0 + stable + nightly julia version would suffice.